pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -245,7 +245,7 @@ spec:
                 default:
                   httpEndpoint: enabled
                   httpProtocolIPv6: disabled
-                  httpPutResponseHopLimit: 2
+                  httpPutResponseHopLimit: 1
```

Comment:
You made this change against main. I think you meant to do it against staging since this is a breaking change

Reply:
Yep, will have to open a new PR I believe.

Reply:
Also, should this change be in the v1 EC2NodeClass?

Reply:
Good catch - will add to new PR here: https://github.com/aws/karpenter-provider-aws/pull/6346